### PR TITLE
Gameplay

### LINKED ISSUE #110: Recovery time should be longer
at least two times longer. The player can take damage twice before correcting his position.

### PR DESCRIPTION
**Pull request by histausse**
fix #110 #93 and #77

### CODE REVIEW

File: README.md
Comment (g33kex) [inline README.md]: Please use an embedded link : 
```
see [here](https://paf.telecom-paris.fr/projets/game-boy-isaac)
```

**Review by g33kex — CHANGES_REQUESTED** [review]
#110 it's fixed, I think it's fine now
#77 fixed too
for #93 I'm not sure this is exactly how it works in the original game. In the game, Isaac can push/be pushed by the enemies, so when he takes damage he gets kinda blocked by them like normal pushable entities, and it happens even in recovery. But for bullet he seems to be knocked back. Maybe for damage just set his speed to 0 for a frame or two... it's a bit awkward to be knocked back and then to be able to walk right through the enemies. But it's a design choice, so it could work. It's your call.

However please use an embedded link in the README, and you should be all set.

**Comment by histausse** [comment]
We don't have the technology to reproduce the behavior of the original game. the point of #93 is to help you escape, so blocking the speed defeats the purpose of it. And actions with repercussion over multiple iterations of the loop are a PATI to implement.

**Review by g33kex — APPROVED** [review]
Prefect!

**Pull request merged**
